universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts

```diff
@@ -4,33 +4,44 @@ import {TopUpProvider} from '../../core/models/TopUpProvider';
 import {getPriceInEther} from './getPriceInEther';
 import {ValueRounder, TokenPricesService, TokenDetails, ETHER_NATIVE_TOKEN, safeDivide} from '@unilogin/commons';
 
+export interface MinimalAmounts {
+  generalMinimalAmount: string;
+  minimalAmountForRevolut?: string;
+};
+
 export const getMinimalAmountForFiatProvider = async (
   paymentMethod: TopUpProvider,
   requiredDeploymentBalance: string,
   tokenPricesService: TokenPricesService,
   currencyDetails = ETHER_NATIVE_TOKEN,
-) => {
+): Promise<MinimalAmounts> => {
   switch (paymentMethod) {
     case TopUpProvider.RAMP: {
+      const fiatCurrency = 'EUR';
       const currencyPriceInEth = await tokenPricesService.getTokenPriceInEth(currencyDetails);
-      const providerMinimalAmountInFiat = '1';
-      const etherPriceInGBP = (await tokenPricesService.getEtherPriceInCurrency('GBP')).toString();
-      const providerMinimalAmount = getPriceInEther(providerMinimalAmountInFiat, etherPriceInGBP);
-      const providerMinimalAmountInToken = safeDivide(providerMinimalAmount, currencyPriceInEth);
+      const providerMinimalAmountInToken = await convertCurrencyToToken('0.6', currencyPriceInEth, fiatCurrency, tokenPricesService);
+      const providerMinimalAmountForRevolutInToken = await convertCurrencyToToken('2', currencyPriceInEth, fiatCurrency, tokenPricesService);
       const requiredDeploymentBalanceAsBigNumber = utils.parseEther(requiredDeploymentBalance);
-      const biggerAmount = bigNumberMax(
-        requiredDeploymentBalanceAsBigNumber,
-        providerMinimalAmountInToken,
-      );
-      return ValueRounder.ceil(utils.formatEther(biggerAmount));
+      const biggerAmount = bigNumberMax(requiredDeploymentBalanceAsBigNumber, providerMinimalAmountInToken);
+      const biggerAmountForRevolut = bigNumberMax(requiredDeploymentBalanceAsBigNumber, providerMinimalAmountForRevolutInToken);
+      return {
+        generalMinimalAmount: ValueRounder.ceil(utils.formatEther(biggerAmount)),
```

Comment:
What do you think about extracting  `getMinimalAmountForRamp`
```ts
const  getMinimalAmountForRamp = (....) =>  { 
     const providerMinimalAmountInToken = await convertCurrencyToToken('0.6', currencyPriceInEth, fiatCurrency, tokenPricesService);
     const biggerAmount = bigNumberMax(requiredDeploymentBalanceAsBigNumber, providerMinimalAmountInToken);
     return ValueRounder.ceil(utils.formatEther(biggerAmountForRevolut));
}
```

Reply:
Done